cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -46,9 +57,16 @@
     protected static final int REPLICATION_PORT = 9091;
     protected static final String REPLICATION_PORT_NAME = "replication";
 
+    protected static final int CLIENT_ENC_PORT = 9093;
+    protected static final String CLIENT_ENC_PORT_NAME = "clientsenc";
```

Comment:
It took me a while to figure out what it means. Is it just me? Wouldn't something like `clientstls` be better? Ot maybe just name them `plaintext` and `ssl` to follow Kafka terminology.

Reply:
Yeah ENC stays for "ENCrypted". I thought a lot about it if using SEC for "SECurity" was better or using TLS as you are suggesting. I have no strong opinion on that so TLS works fine. I'll change everywhere ENC is used and using TLS instead.